geometry_msgs/Point closest_vertex
geometry_msgs/Point[] path